hwilib/commands.py

```diff
@@ -114,7 +114,13 @@ def enumerate(password: str = "") -> List[Dict[str, Any]]:
     for module in all_devs:
         try:
             imported_dev = importlib.import_module('.devices.' + module, __package__)
-            result.extend(imported_dev.enumerate(password)) # type: ignore
+            partial_res = imported_dev.enumerate(password)  # type: ignore
+            # normalize master fingerprint to lowercase
+            for device_dict in partial_res:
+                # if None or other type than string - skip
+                if device_dict['fingerprint'] and isinstance(device_dict['fingerprint'], str):
```

Comment:
not sure if it should raise `ValueError` if master fp is not string

Reply:
or even attempt some further normalization? , i.e. `if isinstance(device_dict['fingerprint'], bytes):  device_dict['fingerprint'] = device_dict['fingerprint'].hex()`